Makefile

```diff
@@ -0,0 +1,35 @@
+.DEFAULT_GOAL := test-all
+
```

Comment:
```suggestion
```

I'm not sure about this default goal. Since the default goal is typically about building (and there's nothing to build in this library), maybe explicitly requiring a target is better than a surprising default goal.